Project.toml

```diff
@@ -17,11 +17,12 @@ ChainRulesCore = "1"
 julia = "^1.0"
 
 [extras]
+AbstractFFTsTestUtils = "9f9ca6b3-60fe-4d4a-946b-987e743140f1"
```

Comment:
I think this won't work since then each of the packages would depend on the other:
```suggestion
```